zproject/urls.py

```diff
@@ -243,6 +243,9 @@
         {'POST': 'regenerate_api_key'}),
     url(r'^users/me/enter-sends$', 'rest_dispatch',
         {'POST': 'change_enter_sends'}),
+    url(r'^users/me/star_user$', 'rest_dispatch',
+        {'GET': 'list_starred_user',
+         'POST': 'mark_user_star'}),
```

Comment:
We are definitely encouraging RESTful style endpoints, and agreed on both the name and that this should be a PUT.  The `_backend` names are largely a legacy thing from when we had multiple "frontend" endpoints with the same implementation; I expect at some point we will drop the `_backend` from them all.  

I added a few notes to https://github.com/zulip/zulip/issues/880 on these points, so we remember to cover them in the documentation.